### PR TITLE
[12.0] Multiple Sales Team

### PR DESCRIPTION
**Pull request by FelipeEnriqueGG**
Sometimes a company has a sale agent in multiple teams. Odoo doesn't allow to add a user to multiple sales teams, this module allow to have one user un multiple sales teams or channels

### CODE REVIEW

**Comment by pedrobaeza** [comment]
cc @rafaelbn

File: sales_team_multi/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    'name': "Multiple sales teams",
+
+    'summary': """
+        it allows to add a user to multiple teams or sales channels.""",
+
+    'author': "BerrySoft, Odoo Community Association (OCA)",
+    'website': "https://github.com/oca",
```
Comment (nikul-serpentcs) [inline sales_team_multi/__manifest__.py]: @FelipeEnriqueGG URL should be `https://github.com/OCA/sale-workflow`

**Review by nikul-serpentcs — COMMENTED** [review]
Nitpicking

**Review by HaraldPanten — APPROVED** [review]
Functional review. Seems to be OK 👍

**Comment by rousseldenis** [comment]
@FelipeEnriqueGG If this is still applicable, could you rebase ?

**Pull request closed without merging**